.github/workflows/python-publish.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish to Pypi
```

Comment:
please remove this file. I have added it in a different [PR](https://github.com/aiplanethub/genai-stack/pull/78)